functions/src/endpoint_paths/event.ts

```diff
@@ -125,4 +126,33 @@ events_app.post('/add', async (req, res) => {
             }
         }
     }
-})
\ No newline at end of file
+
+    // Check that user is not resident in the next page
+})
+
+/**
+ * Get all events available to the user
+ * 
+ * @throws 500 - Server Error
+ */
+events_app.post('/get', async (req, res) => {
```

Comment:
The format that I have been changing things to for a general get request is to use events_app.get instead of events_app.post and to use a path of '/' instead of '/get' because I think the HTTP method makes the '/get' redundant.